src/BgLZ-2-0/Cert/PHR-Client/medmij-bglz-phr-1-1-retrieve-bglz.xml

```diff
@@ -138,14 +138,11 @@
         <nts:include value="assert-authorizationHeader" scope="common"/>
         <nts:include value="assert-request" scope="common"/>
         <nts:include value="assert-responseSearchBundleSuccess" scope="common"/>
-        <action> 
-            <assert> 
-                <description value="Confirm that the returned searchset Bundle contains 1 CarePlan resources. Set to warningOnly because of https://bits.nictiz.nl/browse/MM-1432, please contact kwalificatie@medmij.nl."/>  
-                <direction value="response"/>  
-                <expression value="Bundle.entry.where(resource.is(CarePlan)).count() = 1"/> 
-                <warningOnly value="true"/>
-            </assert> 
-        </action>
+        <nts:include value="assert-returnCount" scope="common">
+            <nts:with-parameter name="resource" value="CarePlan"/>
+            <nts:with-parameter name="count" value="2"/>
+            <nts:with-parameter name="additionalDescription" value="Consists of 1 structured and 1 unstructured CarePlan."/>
+        </nts:include>
```

Comment:
Ik heb hier dus m'n twijfels bij (sorry ik had al m'n comments bij de 'src' moeten zetten, niet bij de 'dev' output.

Reply:
Dit is ook alleen bij BgLZ-2-0 gedaan en niet bij BgLZ-3-0?
Voor 2-0 is sowieso een beetje vaag of er nog onderhoud gedaan moet worden, want er mag niemand op kwalificeren.

Reply:
Idd, dit kan eigenlijk ook gesplitst worden.
Goede vraag wel. Als er niet meer opgekwalificeerd mag worden... kunnen we dat hele mappie dan niet beter verwijderen?

Reply:
Wacht, nee dit klopt wel. Wij leveren nu toch altijd twee varianten uit? ééntje o.b.v de narrative en ééntje zonder. PGO's moeten met beide om kunnen gaan.

Voglens mij staat het zo dus goed. Het gene dat ik gewijzigd hebt zorgt er alleen voor dat je geen warning meer zit als je de testscripts afvuurd. 

Neemt niet weg dat we het hele mapje beter kunnen weghalen als er toch niet meer mee gekwalificeerd wordt.